app/views/alchemy/elements/_legal_editor.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= element_editor_for(element) do |el| %>
```

Comment:
This should not be an Element IMO but rather a menu. I'll be making some menus this afternoon for Fundfinder; they're a new Alchemy Feature.

Reply:
PtW is still on Alchemy 4.1, I think menus were implemented in 4.4.

Upgrading Alchemy should definitely be on the roadmap.

Reply:
Oh yeah